lib/cryptoexchange/exchanges/stake_cube/services/trades.rb

```diff
@@ -17,7 +17,7 @@ def adapt(output, market_pair)
             tr.base      = market_pair.base
             tr.target    = market_pair.target
             tr.type      = trade['type']
-            tr.price     = trade['price']
+            tr.price     = trade['PRICE']
```

Comment:
any reason they are just updating this attribute to be capital case?

Reply:
not sure why, but it's in their DOC as well.